Source/Tenancy/IPerformActionOnAllTenants.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Dolittle.Execution;
+using Dolittle.Tenancy;
+
+namespace Dolittle.Runtime.Tenancy
+{
+    /// <summary>
+    /// Defines a system that perform an action for <see cref="ITenants.All" /> in an <see cref="ExecutionContext" /> with the tenant set for each Tenant.
+    /// </summary>
+    public interface IPerformActionOnAllTenants
```

Comment:
Suggest renaming either the class or the interface to be following the `IFoo` -> `Foo` convention, since this is not used for discovery and typically how we'd name it if there are multiple of these. 
That way you can get rid of the explicit binding for this and the convention will catch it.

Reply:
Interface could be something like : `ITenantsActionPerformer` implementation `TenantsActionPerformer`

Reply:
Yeah, that sounds reasonable

Reply:
Fixed by https://github.com/dolittle-runtime/Runtime/pull/381/commits/efba7f4fccbaf107e0cd31ac1eba6948f06e5c0b